pair-filter/dockerhub_images.json

```diff
@@ -1,2 +1,4 @@
+  "travisci/ci-sardonyx": "packer-1528185911-792f90b\npacker-1528192193-c8fb91e\npacker-1528301903-5b76c82\npacker-1531387074-d9a9e02-dirty\npacker-1531390649-62bb97e\npacker-1531810159-e3f87ae\npacker-1532040386-a02aede\npacker-1533043985-e8975dc\npacker-1538992649-ea2e166\npacker-1539000503-2058f45-dirty\npacker-1539074369-2058f45-dirty\npacker-1539157241-2058f45\npacker-1539248687-2058f45\npacker-1539699952-a2f31db\npacker-1539708047-01f27ab\npacker-1540299711-e74d3c3\npacker-1540467555-c96f8c1\npacker-1540809808-065b8a8\npacker-1540891792-ee23c8a\npacker-1540899313-05eb046\npacker-1540999397-bc6e2f5\npacker-1541062279-3f686ce\npacker-1541197160-c2938f7\npacker-1541445940-e193d27\npacker-1542034427-769f89f\npacker-1542034632-769f89f\npacker-1542036377-e5a593b\npacker-1542096829-107e267\npacker-1542096838-db989b2\npacker-1542104228-d128723\npacker-1542104249-416ffb0\npacker-1545232838-63401b1\npacker-1545232908-63401b1\npacker-1545236977-e8ffc8a\npacker-1545391367-cb6d370\npacker-1546853098-f3203ce\npacker-1546878226-7ce7ab5\npacker-1546978056-2c98a19\npacker-1547137860-b245eb0\npacker-1547195794-5f6e95a\npacker-1547196294-1546a5a\npacker-1547196319-5f6e95a\npacker-1547455648-2c98a19\npacker-1548200532-2c98a19\npacker-1548413020-2e42d95\npacker-1548671919-2e42d95\npacker-1548686831-2313a9e\npacker-1549881206-387f377\npacker-1552557266-f909ac5\npacker-1553529172-f909ac5\npacker-1553530531-f909ac5\npacker-1554885359-f909ac5\npacker-1558622123-f909ac5\npacker-1558623664-f909ac5\npacker-1564753982-0c06deb6\npacker-1576238197-60d50014"
+}
```

Comment:
Does the component of `PairFilter` that uses this need to be changed also?

Reply:
Also, we need to split the entry for each key by `\n`.

Reply:
It does not, it still works successfully with the change. FYI I've also added another necessary file into `reproducer`

Reply:
The current implementation uses REGEX to parse through the value, [here](https://github.com/BugSwarm/bugswarm/blob/dockerhub-images-format-fix/pair-filter/pair_filter/image_chooser.py#L143). If you want it changed I can do that to utilize `split` for the entry, I will have to refactor a few things.

Reply:
@octaviocorona I am thinking that the file should be of the format:
```
{
  "travisci/ci-garnet": [
     "packer-1469045719",
     "packer-1469063753",
      ...
    ],
  ...
}
```
It seems better than a long string of every Docker image appended together. What do you think?

Reply:
I completely agree.